lib/options/options.vue

```diff
@@ -78,15 +117,52 @@ $secondary: $light-gray;
 
 @import '../assets/common';
 
+body, html, main, .content {
```

Comment:
I deleted this locally and it removed the scroll-bar on the window which was too much. It may have side-effects, but I didn't see any. 
So, maybe you could try this.

Reply:
Yes it works. Thanks.